From 666043aa0b0708e7672ed3a36ce70d859c5b5af9 Mon Sep 17 00:00:00 2001
From: Namit Solanki <nsolanki@codeaurora.org>
Date: Wed, 6 Mar 2019 18:38:28 +0530
Subject: [PATCH] glibc:2.28:include sysmacros.h for major and minor calls.

* Since glibc-2.26, major and minor calls have been defined
  in sysmacros.h.
* Include sysmacros.h from sys for references to major and minor
  macro calls.

Change-Id: Ic1794f34b895059499b3d1ef8b7e2391112dce75
---
 src/launcher-util.c | 1 +
 src/logind-util.c   | 1 +
 src/weston-launch.c | 1 +
 3 files changed, 3 insertions(+)

diff --git a/src/launcher-util.c b/src/launcher-util.c
index 004b705..558e3a5 100644
--- a/src/launcher-util.c
+++ b/src/launcher-util.c
@@ -34,6 +34,7 @@
 #include <signal.h>
 #include <sys/socket.h>
 #include <sys/types.h>
+#include <sys/sysmacros.h>
 #include <sys/stat.h>
 #include <sys/uio.h>
 #include <sys/ioctl.h>
diff --git a/src/logind-util.c b/src/logind-util.c
index 006a1ae..9eebf6b 100644
--- a/src/logind-util.c
+++ b/src/logind-util.c
@@ -39,6 +39,7 @@
 #include <sys/ioctl.h>
 #include <sys/signalfd.h>
 #include <sys/stat.h>
+#include <sys/sysmacros.h>
 #include <systemd/sd-login.h>
 #include <unistd.h>
 
diff --git a/src/weston-launch.c b/src/weston-launch.c
index b8dfb17..5ec74ff 100644
--- a/src/weston-launch.c
+++ b/src/weston-launch.c
@@ -37,6 +37,7 @@
 #include <getopt.h>
 
 #include <sys/types.h>
+#include <sys/sysmacros.h>
 #include <sys/ioctl.h>
 #include <sys/stat.h>
 #include <sys/wait.h>
-- 
1.9.1

